### PR TITLE
update link design

### PR DESCRIPTION
**Pull request by jeaunrg**
[ #37 ] This branch contains any design improvement.

### CODE REVIEW

**Comment by jeaunrg** [comment]
I didnt find a better/easier way to create a link with arrows inside, the way I did it is weird but it works well. If you have any idea that can be readily understood, Im in!

File: src/view/graph.py [hunk 1]
```diff
@@ -76,31 +66,15 @@ class Junction(QtWidgets.QGraphicsPolygonItem):
     color: QColor
 
     """
-    def __init__(self, node, type, size=5, color=QtGui.QColor(0, 150, 0)):
+    def __init__(self, node):
```
Comment (Hboni) [inline src/view/graph.py:69]: Update `Junction` docstring is needed then

File: src/view/graph.py [hunk 2]
```diff
@@ -175,53 +149,22 @@ def mouseReleaseEvent(self, event=None):
...
-
         Return
         ------
         junction: Junction
         """
-        junction = Junction(self, type, size, color)
+        junction = Junction(self)
         self.positionChanged.connect(junction.updateLinkPos)
-        self.sizeChanged.connect(lambda: junction.setPos(*self.getJunctionRelativePosition(type)))
+        self.sizeChanged.connect(lambda: junction.setPos(self.width()/2, self.height()/2))
```
Comment (Hboni) [inline src/view/graph.py:162]: To be clearer you can use @property decorator to use a specific name for that operation like:

    @property
    def mid_pos(self):
        return self.width()/2, self.height()/2

and then call it here something like that:

    self.sizeChanged.connect(lambda: junction.setPos(*self.mid_pos))

If you prefer having the operation here it is ok also ;)

**Review by Hboni — CHANGES_REQUESTED** [review]

File: src/view/graph.py (hunk 2 above)
Comment (jeaunrg) [reply src/view/graph.py:162]: ok I never used this decorator!

**Comment by jeaunrg** [comment]
@Hboni I succeed in creating the link arrow like we said! 
arrow

The arrow can be customized as you wish.
I also removed junction which where useless.

File: src/view/graph.py (hunk 2 above)
Comment (Hboni) [reply src/view/graph.py]: It can be useful to have code easier to understand 😉

**Comment by Hboni** [comment]
Is it normal that the arrow is a little bit detached from the node?

**Comment by jeaunrg** [comment]
> Is it normal that the arrow is a little bit detached from the node?

Yes there is a parameter 'space' for the arrow to control this detachment

File: src/view/graph.py [hunk 3]
```diff
@@ -374,11 +384,18 @@ def bind(self, parent, child):
         parent, child: Node
             nodes to visually bind
         """
-        jout = parent.addJunction()
-        jin = child.addJunction()
-        link = Link(jout, jin,
-                    GRAPH_PARAMETERS['lineWidth'],
-                    GRAPH_PARAMETERS['lineColor'])
+        # jout = parent.addJunction()
+        # jin = child.addJunction()
```
Comment (Hboni) [inline src/view/graph.py:388]: clean

Comment (jeaunrg) [reply src/view/graph.py]: done

**Review by Hboni — APPROVED** [review]

**Pull request merged**